pcsx2/CDVD/CDVD.cpp

```diff
@@ -958,7 +958,7 @@ u8 monthmap[13] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
 
 void cdvdVsync() {
 	cdvd.RTCcount++;
-	if (cdvd.RTCcount < ((gsRegionMode == Region_NTSC) ? 60 : 50)) return;
+	if (cdvd.RTCcount < ((gsVideoMode == NTSC) ? 60 : 50)) return;
```

Comment:
> Sorry, I actually meant the RTC Clock. Is it used for any important operations ? as there's a bug for progressive mode on RTCcount which increments the RTC Clock.

the RTC clock is used for Katamari Damacy and MGS3 will refuse to boot without it :p

I don't think it's bugged, not very much anyway. I don't think any game with progressive mode needs it, but that will probably just make the clock run slow (or fast)